Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -622,6 +622,18 @@ def test_create_subprocess_shell_text_mode_fails(self):
         self.loop.run_until_complete(execute())
 
 
+    def test_create_subprocess_exec_with_path(self):
+        async def execute():
+            from pathlib import Path
+            p = await subprocess.create_subprocess_exec(
+                Path(sys.executable), '-c', 'pass')
+            await p.wait()
+            p = await subprocess.create_subprocess_exec(
+                sys.executable, '-c', 'pass', Path('.'))
+            await p.wait()
+
+        self.loop.run_until_complete(execute())
```

Comment:
please add `self.assertIsNone(loop.run_until_complete(...))` check to emphasize that we checked that the test passed correctly.